eng/bomgenerator/inputdir/pom.xml

```diff
@@ -0,0 +1,380 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
The contents in `inputdir` and `outputdir` are just helper files we use to generate the BOM. We don't need to include them